pkg/ccl/backupccl/restore_processor_planning.go

```diff
@@ -188,6 +188,7 @@ func distRestore(
 			NumEntries:           int64(numImportSpans),
 			NumNodes:             int64(numNodes),
 			UseSimpleImportSpans: useSimpleImportSpans,
+			JobID:                jobID,
```

Comment:
This isn't mentioned in the commit message. What's the impact of this being missing?

Reply:
Oops I'll add it to the commit message. The consequence was log lines on remote processors not having the JobID in their tags.